src/Microsoft.DotNet.Build.Tasks/PackageFiles/versioning.targets

```diff
@@ -16,6 +16,68 @@
   <Import Project="$(BuildNumberTarget)" Condition="Exists('$(BuildNumberTarget)')" />
 
   <!-- #################################### -->
+  <!-- Version Info -->
+  <!-- #################################### -->
+
+  <ItemGroup>
+    <VersionInfoFile Include="$(BinDir)VersionInfo.json"/>
```

Comment:
It will be rewritten on a Clean/Rebuild because it is added to FileWrites. If the versions are slightly off locally that isn't a problem.